### PR TITLE
[YOLO] Update README with the new release adding guide

### PR DESCRIPTION
**Pull request by lkanggithub**
## Summary
Add a section in README.md about how to add  a new release. The content is based on the following wiki pages. I feel it would be better to document this in the repository.
https://datarobot.atlassian.net/wiki/spaces/RAPTOR/pages/1419247790/Publishing+DRUM+to+PyPi
https://datarobot.atlassian.net/wiki/spaces/RAPTOR/pages/1416691967/Install+drop-in+environments+on+staging+production

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,6 +137,56 @@ Check for `generate*` scripts in [./tests/fixtures/drop_in_model_artifacts](./te
...
+
+3. Publish the new release in PyPI with the Jenkins Job (https://jenkins.hq.datarobot.com/job/Release_DRUM_PyPI/)
+
+   Build parameters:
+   - RELEASE_TAG: The new release number.
+   - PYPI_PASSWORD: If you don’t have PYPI_PASSWORD, reuse another successful job run by rebuilding it (password will be filled in).
+
+4. Update dr_requirements.txt in the testing under DataRobot repo (https://github.com/datarobot/DataRobot)
+
+   Update the dr_requirements.txt in functional testings (e.g., `./tests/raptor_lake_templates/environments/python_base/dr_requirements.txt`) and make sure all testings pass.
```
Comment (lkanggithub) [inline README.md:166]: Besides the testing in datarobot-user-blueprint repo, I think we should also verify the latest release with the functional testing in DataRobot repo before updating the drop-in env in Staging/Production. 
Your thought?
@4dahalibut @yakov-g

File: README.md [hunk 2]
```diff
@@ -137,6 +137,56 @@ Check for `generate*` scripts in [./tests/fixtures/drop_in_model_artifacts](./te
 Model examples in [./model_templates](./model_templates) are also used in functional testing. In the most cases, artifacts for those models are the same as in the [./tests/fixtures/drop_in_model_artifacts](./tests/fixtures/drop_in_model_artifacts) and can be simply copied accordingly.
 If artifact for model template is not in the [./tests/fixtures/drop_in_model_artifacts](./tests/fixtures/drop_in_model_artifacts), check template's README for more instructions.
 
+### Adding a new release
```
Comment (4dahalibut) [inline README.md:140]: Can you make a new file called `RELEASE.md` and put this in there maybe?

File: README.md [hunk 3]
```diff
@@ -137,6 +137,56 @@ Check for `generate*` scripts in [./tests/fixtures/drop_in_model_artifacts](./te
...
+- Run Build_Environments_Installer jenkins job (https://jenkins.hq.datarobot.com/job/Build_Environments_Installer/). After the build, the tarball with all the environments will be generated in S3 (available in the log).
+
+  Build parameters:
+  - CUSTOM_MODEL_INSTALLER_VERSION: Set with the latest version.
+  - CUSTOM_MODEL_INSTALLER_ENV: staging
+- Run Release_Public_Environments_Staging jenkins job (https://jenkins.hq.datarobot.com/job/Release_Public_Environments_Staging/)
+
+  This will install drop-in environments from the built tarball in the Staging.
+
+6. Update drop-in environments in Production (**US/EU**)
```
Comment (4dahalibut) [inline README.md:178]: I think you should draw attention to the fact that this script has to be run twice

**Review by 4dahalibut — CHANGES_REQUESTED** [review]
Small improvements

**Review by jmbannon — CHANGES_REQUESTED** [review]
I think we are giving out too much information to the public, I think you should check with security and make sure they are okay with this. If not, the branch and PR should be force deleted to remove from the git history

**Comment by lkanggithub** [comment]
Close this pr. A new page https://datarobot.atlassian.net/wiki/spaces/RAPTOR/pages/4982014178/Create+a+new+release+of+DRUM was created to document the process.

**Pull request closed without merging**